Makefile

```diff
@@ -0,0 +1,25 @@
+BIN ?= cchat-gtk
+GO ?= go
+
+PREFIX ?= /usr/local
+BINDIR ?= $(PREFIX)/bin
+DATADIR ?= $(PREFIX)/share
+
+cchat-gkt: 
```

Comment:
There's a typo here; shouldn't this be `cchat-gtk`?

Reply:
Sorry it was my mistake